fastai/tabular/learner.py

```diff
@@ -1,17 +1,21 @@
-# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/43_tabular.learner.ipynb (unless otherwise specified).
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/nbs/43_tabular.learner.ipynb (unless otherwise specified).
 
 __all__ = ['TabularLearner', 'tabular_learner']
 
 # Cell
 from ..basics import *
 from .core import *
 from .model import *
+from .data import *
```

Comment:
keep the import for data in a separate cell so it isn't imported into the lib.  
![image](https://user-images.githubusercontent.com/7451178/154828712-4d7c53f1-6089-48d0-9b56-e17dad5e844c.png)
If you notice in the original version, the `from fastai.tabular.data import *` line doesn't have `#export` so it won't be added to the .py file but it looks like you have maybe combined those cells.

Reply:
if I don't include `from fastai.tabular.data import *` as part of the `#export` call I will have to do `fastai.tabular.data.TabularDataloaders` when indicating the `type` for `dls` in `tabular_learner` . By including `from fastai.tabular.data import *`  in the `#export`,  I can do `TabularDataloaders`.

Reply:
Feel free to keep this in so typing works :)